packages/core/src/components/application-header/application-header.scss

```
@@ -102,6 +103,9 @@
     }
 
     .application-name {
+      @include fonts.typography-body-lg;
+      margin: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)